3_javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,68 @@
+
+const ApiKey = `https://open.er-api.com/v6/latest/EUR`;
+
+
+
+let toSelectList = document.createElement("select");
+let fromSelectList = document.createElement("select");
+let rates;
+
+fetch(ApiKey)
+    .then(Response => Response.json())
```

Comment:
In JavaScript, variable names always start with a small letter. It won't affect the code working or not working, but it will help others understand your code easier :)

See https://www.w3schools.com/js/js_conventions.asp

Reply:
yes thanks a lot for  helping me .